sound/soc/intel/common/soc-acpi-intel-ptl-match.c

```diff
@@ -305,6 +349,12 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_ptl_sdw_machines[] = {
 		.drv_name = "sof_sdw",
 		.sof_tplg_filename = "sof-ptl-rt715-rt711-rt1308-mono.tplg",
 	},
+	{
+		.link_mask = BIT(3),
+		.links = ptl_cs42l43_l3,
+		.drv_name = "sof_sdw",
+		.sof_tplg_filename = "sof-ptl-cs42l43.tplg",
+	},
```

Comment:
any specific reason to put this here? This doesn't seem to follow any order that I understand. It mixes Bit(3) and Bit(0) definitions. Maybe best to have a clearer set of conventions on where entries get added

Reply:
Thanks for the review Pierre!!

Assumed mockup-test entries make it 1st
& then follow the usual <platform>_<codec>_<link#> convention